tests/plugins/test_compress.py

```diff
@@ -34,12 +34,13 @@ class X(object):
 
 
 class TestCompress(object):
-    @pytest.mark.parametrize('method, load_exported_image, extension', [
-        ('gzip', False, 'gz'),
-        ('lzma', False, 'xz'),
-        ('gzip', True, 'gz'),
+    @pytest.mark.parametrize('method, load_exported_image, give_export, extension', [
+        ('gzip', False, True, 'gz'),
+        ('lzma', False, False, 'xz'),
+        ('gzip', True, True, 'gz'),
+        ('gzip', True, False, 'gz'),
```

Comment:
Since you're editing this test, could you add a parameter for an unsupported compression format test here?